contracts/Market.sol

```diff
@@ -389,26 +385,52 @@ contract Market is IMarket, Ownable, ERC721 {
 		return _bets[index].payout;
 	}
 
-
 	function _payout(uint256 index, uint8 result) internal virtual {
-		if (result == SCRATCHED) {
-			uint256 lay = _bets[index].payout - _bets[index].amount;
+		uint256 payout = _bets[index].payout;
+		uint256 amount = _bets[index].amount;
+		uint256 loan = payout - amount;
+
+		// Deduct from total exposure first
+		_totalExposure -= loan;
+
+		if (result == SCRATCHED) {				
 			// Transfer the bet amount to the owner of the NFT
-			IERC20(_vault.asset()).transfer(ownerOf(index), _bets[index].amount);
-			// Transfer the lay back to the vault
-			IERC20(_vault.asset()).transfer(address(_vault), lay);
-		} else {
-			address recipient = result == WINNER ? ownerOf(index) : address(_vault);
-			uint256 amount = _bets[index].payout;
+			IERC20(_underlying).transfer(ownerOf(index), amount);
```

Comment:
i believe we should be storing this in a var in constructor:
```
IERC20 private immutable _underlying;

// ...

_underlying = IERC20(vault.asset());

// ...

_underlying.transfer(ownerOf(index), amount);

// etc...
```

Reply:
saves on reinitialising afaik

Reply:
Fair, and follows the pattern.